src/model.ts

```diff
@@ -21,6 +21,8 @@ export class Model {
 	readonly $culture: CultureInfo;
 
 	readonly entityRegistered: EventSubscriber<Model, EntityRegisteredEventArgs>;
+	readonly afterPropertySet: Event<Entity, EntityChangeEventArgs>;
+	readonly listChanged: Event<Entity, EntityChangeEventArgs>;
```

Comment:
These two properties should just be `EventSubscriber`, so external users don't get access to the `publish` method.

Reply:
Change made.